frontend/src/views/web/error-pages/error-404.html

```diff
@@ -1,15 +1,29 @@
 <div class="fof-wrapper">
   <div class="fof-error-container">
-    <div class="fof-error">
-      Error 404
-    </div>
-
-    <div class="fof-message">
-      Woops. Looks like this page doesn't exist.
+    <div class="fof-grad">
+	  	<div class="fof-stars" id="star-1"></div>
+	  	<div class="fof-stars" id="star-2"></div>
+	  	<div class="fof-stars" id="star-3"></div>
+	  	<div class="fof-stars" id="star-4"></div>
+	  	<div class="fof-stars" id="star-5"></div>
+	  	<div class="fof-stars" id="star-6"></div>
+	  	<div class="fof-stars" id="star-7"></div>
+	  	<div class="fof-stars" id="star-8"></div>
+	  	<div class="fof-stars" id="star-9"></div>
+	  	<div class="fof-stars" id="star-10"></div>
+	  	<div class="fof-stars" id="star-11"></div>
+	  	<div class="fof-stars" id="star-12"></div>
+	  	<div class="owl-pic"></div>
+    	<div class="fof-error">
+    		404
```

Comment:
@Ram81 all the checks will pass if you change it to `Error 404` or change the `test_views.py` that you changed earlier.